config/salesforce.spc

```diff
@@ -1,9 +1,27 @@
 connection "salesforce" {
   plugin = "salesforce"
 
-  url       = "YOUR_SALESFORCE_URL"
-  user      = "YOUR_SALESFORCE_USERNAME"
-  password  = "YOUR_SALESFORCE_PASSWORD"
-  token     = "YOUR_SALESFORCE_TOKEN"
-  client_id = "YOUR_SALESFORCE_CLIENT_ID"
+  # Salesforce instance URL, e.g., "https://na01.salesforce.com/"
+  # url = "YOUR_SALESFORCE_URL"
+
+  # Salesforce account name
+  # user = "YOUR_SALESFORCE_USERNAME"
+
+  # Salesforce account password
+  # password = "YOUR_SALESFORCE_PASSWORD"
+
+  # Security token, could be omitted if Trusted IP is configured
+  # token = "YOUR_SALESFORCE_TOKEN"
+
+  # Salesforce client ID from the connected app
+  # client_id = "YOUR_SALESFORCE_CLIENT_ID"
+
+  # List of Salesforce object names to generate additional tables for
+  # This argument only accepts exact Salesforce standard and custom object names, e.g., AccountBrand, OpportunityStage, CustomApp__c
+  # For a full list of standard object names, please see https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_objects_list.htm)
+  # All custom object names should end in "__c", following Salesforce object naming
+  # tables = ["AccountBrand", "OpportunityStage", "CustomApp__c"]
```

Comment:
```suggestion
  # objects = ["AccountBrand", "OpportunityStage", "CustomApp__c"]
```

Should this be `objects` instead of `tables` maybe? Salesforce refers to them as `objects` consistently in the API, I think UI as well (but please correct me if I'm wrong).

If we do decide to make this change, we should make sure that code, docs, etc., are all updated to reflect this as well.